src/Arcus.Messaging.Tests.Workers/MessageHandlers/OrdersMessageHandler.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Arcus.EventGrid.Publishing.Interfaces;
+using Arcus.Messaging.Abstractions;
+using Arcus.Messaging.Pumps.ServiceBus;
+using Arcus.Messaging.Tests.Core.Events.v1;
+using Arcus.Messaging.Tests.Core.Messages.v1;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+
+namespace Arcus.Messaging.Tests.Workers.MessageHandlers
+{
+    public class OrdersMessageHandler : IAzureServiceBusMessageHandler<Order>
```

Comment:
Thinking out loud - If I were to not care about the Service Bus message context, can I still use `IMessageHandler<Order>`?

That would be very good to have imo. Reduces lockin.

Reply:
For clarity, I see value for both scenarios

Reply:
Well, the `IMessageHandler` uses the context in it's signature. So... we would have to create an different `IMessageHandler` interface with maybe an "ignored" message context so we can also include that in the filter.
When processing the message, the context would then be empty. or a `null`-type.

Reply:
In any case; different PR. Can you make an issue for this?

Reply:
See other comment, message context will always be there but it will just be specific to SB or the generic model.